pandas/tests/indexing/multiindex/test_getitem.py

```diff
@@ -87,7 +86,8 @@ def test_series_getitem_returns_scalar(
         (lambda s: s[(2000, 3, 4)], KeyError, r"^\(2000, 3, 4\)$"),
         (lambda s: s.loc[(2000, 3, 4)], KeyError, r"^\(2000, 3, 4\)$"),
         (lambda s: s.loc[(2000, 3, 4, 5)], IndexingError, "Too many indexers"),
-        (lambda s: s.__getitem__(len(s)), KeyError, ""),  # match should include len(s)
+        (lambda s: s.__getitem__(len(s)), KeyError, ""),
+        # match should include len(s)
```

Comment:
Is there a reason you are doing this and removed the line above?

Reply:
This was changed by accident. Sorry for missing it.

Reply:
Should I revert those changes in this PR?

Reply:
yes please

Reply:
Thank you for pointing this out. Requested changes were implemented.